### PR TITLE
[CI] Fix Vitis-AI tests when USE_VITIS_AI flag set to OFF 

### PR DESCRIPTION
**Pull request by anilmartha**
Hi @leandron,

We have fixed the vitis-ai test failure [issue ](https://github.com/apache/tvm/issues/10757)when running with USE_VITIS_AI OFF flag. Could you please review and merge it?

### CODE REVIEW

**Comment by leandron** [comment]
Hi @anilmartha, thanks for the PR. It looks very promising in fixing https://github.com/apache/tvm/pull/10802.

Do you mind renaming it so that is it more clear for other people not involved in the original issue, what problem this PR is fixing? I would also recommend putting a `[CI]` tag, so that it looks more like other PRs.

cc @ashutosh-arm @driazati @lhutton1 for reviews

**Comment by anilmartha** [comment]
@leandron changed the title as per your suggestions.

File: tests/python/contrib/test_vitis_ai/infrastructure.py [hunk 1]
```diff
@@ -59,7 +59,7 @@ def visit_call(self, call):
 
 def skip_test():
```
Comment (ashutosh-arm) [inline tests/python/contrib/test_vitis_ai/infrastructure.py:60]: Overall LGTM! I see that tvm has more or less a standard way of defining require_*() functions in this file and using those functions to skip the tests. Maybe you could define a similar function for vitis_ai.
https://github.com/apache/tvm/blob/main/python/tvm/testing/utils.py

File: tests/python/contrib/test_vitis_ai/test_vitis_ai_codegen.py [hunk 2]
```diff
@@ -54,6 +54,8 @@ def set_func_attr(func, compile_name, symbol_name):
 )
 def test_conv2d(dpu_target):
```
Comment (driazati) [inline tests/python/contrib/test_vitis_ai/test_vitis_ai_codegen.py:55]: Can you get rid of `skip_test` and use pytest’s skipIf decorator so the skips are properly reported?

https://docs.pytest.org/en/7.1.x/how-to/skipping.html#id1

Comment (anilmartha) [reply tests/python/contrib/test_vitis_ai/test_vitis_ai_codegen.py]: @driazati 
I replaced skip_test with pytest's skipif. Could you have a another look?

**Review by manupa-arm — APPROVED** [review]
This looks good for now!

Though, I would suggest to implement a requires function to be consistent with others (See below) -- possibly for a follow up ? lmk

https://github.com/apache/tvm/blob/47ab54538fb83dc836f6516ed63870495df56253/python/tvm/testing/utils.py#L900-L951

**Comment by manupa-arm** [comment]
Thanks @anilmartha! LGTM!

**Review by driazati — APPROVED** [review]

File: tests/python/contrib/test_vitis_ai/test_vitis_ai_codegen.py (hunk 2 above)
Comment (driazati) [reply tests/python/contrib/test_vitis_ai/test_vitis_ai_codegen.py]: looks good, thanks!

**Comment by manupa-arm** [comment]
Thanks @anilmartha @ashutosh-arm @driazati !

**Pull request merged**